### PR TITLE
96: + added flow of temp saving

### PR DESCRIPTION
**Pull request by nazar-kotiukha**
added flow of temp saving with mocked getting data from db

### CODE REVIEW

File: backend/Squirrel.Core/Squirrel.Core.BLL/Services/DBStructureSaverService.cs [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        {
+            Id = changeRecord.UniqueChangeId.ToString(),
+            ContentType = "application/json",
+            Content = contentBytes
+        };
+
+        var containerName = _configuration["UserDbChangesBlobContainerName"];
+
+        await _blobStorageService.UploadAsync(containerName, blob);
+    }
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.Core/Squirrel.Core.BLL/Services/DBStructureSaverService.cs]: Can you, please move logic of saving to SQL service?

File: backend/Squirrel.Core/Squirrel.Core.WebAPI/Controllers/ChangeRecordsController.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+    public ChangeRecordsController(IChangeRecordService changeRecordService)
+    {
+        _changeRecordService = changeRecordService;
+    }
+
+    [HttpPost]
+    public async Task<ActionResult<Guid>> AddChangeRecord()
+    {
+        return Ok(await _changeRecordService.AddChangeRecordAsync());
```
Comment (Anatolii-Ivanov) [inline backend/Squirrel.Core/Squirrel.Core.WebAPI/Controllers/ChangeRecordsController.cs:22]: Not sure if we need this method

File: backend/Squirrel.Core/Squirrel.Core.BLL/Services/DBStructureSaverService.cs (hunk 1 above)
Comment (Anatolii-Ivanov) [reply backend/Squirrel.Core/Squirrel.Core.BLL/Services/DBStructureSaverService.cs]: I meant UI sends request to WebAPI, webAPI sends request for saving to SQL service (in general webapi shouldn't use blobStorage (except user photos))

**Review by Anatolii-Ivanov — DISMISSED** [review]

**Review by alinaSielina — DISMISSED** [review]

**Comment by alinaSielina** [comment]
fix conflicts please

**Review by alinaSielina — DISMISSED** [review]

**Review by alinaSielina — DISMISSED** [review]

**Comment by alinaSielina** [comment]
Resolve conflicts please

**Review by alinaSielina — APPROVED** [review]

**Review by MaxShelestiuk — APPROVED** [review]

**Pull request merged**